hosted_control_planes/hcp-deploy/hcp-deploy-virt.adoc

```diff
@@ -57,7 +57,7 @@ include::modules/hcp-virt-create-hc-console.adoc[leveloffset=+2]
 [role="_additional-resources"]
 .Additional resources
 
-* To create credentials that you can reuse when you create a hosted cluster with the console, see link:https://docs.redhat.com/en/documentation/red_hat_advanced_cluster_management_for_kubernetes/2.12/html/clusters/cluster_mce_overview#creating-a-credential-for-an-on-premises-environment[Creating a credential for an on-premises environment].
+* To create credentials that you can reuse when you create a hosted cluster with the console, see link:https://docs.redhat.com/en/documentation/red_hat_advanced_cluster_management_for_kubernetes/2.12/html/clusters/cluster_mce_overview#creating-a-credential-for-an-on-premises-environment[Creating a credential for an on-premise environment].
```

Comment:
I think this should stay as "on-premises" because it's referencing a title that uses that term.